package.json

```diff
@@ -87,11 +87,20 @@
       {
         "command": "mesonbuild.restartLanguageServer",
         "title": "Meson: Restart Language Server"
+      },
+      {
+        "command": "mesonbuild.selectRootDir",
+        "title": "Meson: Select project root directory"
```

Comment:
Can you make this command title case similar to the others?

Reply:
Should this maybe just be "Select Project"?

Reply:
I feel like "Project" is probably better just because our Memento is `mesonbuild.hasProject`. Plus, I feel like thinking in terms of root directories is a little too low level.